converter.go

```diff
@@ -316,6 +316,9 @@ func stringToValue(
 		}
 		loc := Location(int(offset) - 1440)
 		tt := time.Unix(sec, nsec)
+		if loc == nil {
+			loc = time.Now().Location()
+		}
```

Comment:
should these just be time.Location?  i.e. avoid the Now() call.

Reply:
afaik time.Location{} if not generated by time.Now() is an empty struct which will panic without the name string

Reply:
`loc` is initialized two lines above - so this condition is always false, am I right?